tests/ppx/react/spreadProps.res

```diff
@@ -0,0 +1,23 @@
+@@jsxConfig({version:4, mode: "classic"})
+let c0 = <A x="x" {...p} />
+
+// ignore second one
+let c0 = <A x="x" {...p0} {...p1} />
```

Comment:
Shouldn't this be an error?

Reply:
That would be more nicer, because the second `p1` would be ignored anyway.

Reply:
https://github.com/rescript-lang/syntax/pull/645/commits/afdd64b00fd8412856fa065c57d7052020093ae8